gem/spec/fixtures/earlgrey_yaml/earlgrey_gem.yml

```diff
@@ -0,0 +1,4 @@
+TeacherUITests:
```

Comment:
which yml will be used for the end user? can you give an example?

Reply:
I added earlgrey_gem.yml to EarlGreyExample.